src/shared/codelens/localLambdaRunner.ts

```diff
@@ -493,12 +483,27 @@ export const invokeLambdaFunction = async (params: {
         if (params.onWillAttachDebugger) {
             await params.onWillAttachDebugger()
         }
+
         await attachDebugger({
-            channeLogger: params.channelLogger,
             debugConfig: params.debugConfig,
-            telemetryService: params.telemetryService,
-            startInvokeMillis: startInvokeTime.getTime(),
-            runtime: params.runtime,
+            maxAttempts: MAX_DEBUGGER_ATTEMPTS,
+            channelLogger: params.channelLogger,
+            onRecordAttachDebuggerMetric: (
+                attachResult: boolean | undefined, attempts: number, attachResultDate: Date
+            ): void => {
+                recordAttachDebuggerMetric({
```

Comment:
`recordAttachDebuggerMetric` appears to be the same in each case. Consider making this the default implementation so it doesn't need to be specified unless non-standard behavior is desired.

Reply:
We will unify the `invokeLambdaFunction` calls in the future, reducing the duplication.
We'd have to pass telemetryService, startInvokeTime, and runtime into attachDebugger in order to make a default implementation, which doesn't seem ideal. I didn't come up with a way of avoiding that.

Reply:
Makes sense.